git-refresher/viewing-file-github.qmd

```diff
@@ -0,0 +1,28 @@
+---
+title: "Viewing a file on GitHub"  
+---
+
+To view a file on your repo:
+
+1. Ensure you are still on the GitHub site for the github-sandbox repository. 
+
+2. Navigate to the 'code' tab at the top of the page.
+
+![](images/repo_code_tab.png){fig-alt="Screenshot of the git-academy-sandbox repository on GitHub, highlighting the 'Code' tab."}
+
+3. Find and select the branch you previously created i.e. `<github_id>/main`. You can do this by clicking on the branches button and selecting your branch, or using the branch drop-down as below to click on your branch: 
+
+
+![](images/viewing_branches_drop_down.png){fig-alt="Screenshot of the git-academy-sandbox repository on GitHub, displaying the branches drop-down menu with 'main' selected."}
+
+4. Once you have selected your branch, the drop-down will change to show your branch name as below: 
+
+![](images/branch_highlighted.png){fig-alt="Screenshot of the git-academy-sandbox repository on GitHub, displaying the branches drop-down menu with 'lsnaathorst1/main' selected."}
+
+5. Click on a file to view the most up-to-date synced code for the branch you have selected.
+
+6. Once you have clicked on a file, you can navigate through different files, change branches or search for a file using Files pane.
+
+![](images/files_pane.png){fig-alt="Screenshot of the git-academy-sandbox repository on GitHub, displaying the files pane as seen when viewing files in a branch."}
+
+7. You will need to switch to the code tab to view and edit the raw contents of the file. The preview tab will show you a preview for supported files like Markdown and Quarto and the blame tab shows line by line revision history, tracking who made each change and when.
```

Comment:
"switch to the code tab" is a little bit confusing that there's a code tab along the top tab-set and then a code tab for each file tab-set. Could maybe just provide a line to give context at the start of this paragraph along the lines of 

> Each file will have a sub-set of the following views available to select above it: preview, code and blame.

Then maybe switch the language in the rest of 7 from "tab" to "view" to help differentiate from the overall Code tab.